app/assets/stylesheets/users.scss

```diff
@@ -267,34 +267,30 @@
 
   input {
     border: 0;
-    border-left: 1px solid $form-border-grey;
     border-radius: 0;
     font-size: 16px;
     margin: 0;
     padding: 7px 10px 7px 35px;
-    transition: .2s ease-in-out;
-
-    &:focus {
-      border-color: $primary-green;
-      transition: .2s ease-in-out;
-    }
   }
 }
 
-.users-circuits-search {
+.icon-search-align {
   align-items: center;
   display: flex;
   flex-wrap: nowrap;
   justify-content: center;
-  padding-top: 20px;
+  padding-top: 30px;
 }
 
-.fa.fa-search.user-circuits-search-icon {
-  color: $fa-search-grey;
+.user-circuits-search-icon {
   margin-left: 28px;
   margin-right: -28px;
   z-index: 2;
 
+  &:hover {
```

Comment:
Changes look great but what is the need of making search bar green on hover?

Reply:
To match search icon with CircuitVerse theme(which is green), I added hover effect on it

Reply:
I don’t think that’s necessary. You can remove it.

Reply:
I'm sorry I was late in answering, I'll remove it right away.